cricPredict/User_profile/views.py

```diff
@@ -47,11 +99,11 @@ def form_valid(self, form):
             return HttpResponse("Invalid Username or Password")
 
     def form_invalid(self, form):
+        print(form.errors.as_json())
         return render(self.request, 'login.html', {'form': form})
```

Comment:
there shouldn't be any pages rendered by Django, now that we are only using the APIs